packages/documentation/src/stories/utilities/opacity/opacity.stories.ts

```diff
@@ -31,7 +31,7 @@ export const Opacity: Story = {
   render: (args: Args) => {
     return html`
       <div class="opacity-${args.opacity}">
-        <div class="opacity-content">Opacity ${args.opacity}%</div>
+        <div>Opacity ${args.opacity}%</div>
```

Comment:
Having the least amount of code in the snippet is better for understanding:

```suggestion
        Opacity ${args.opacity}%
```